cli/src/bin/main.rs

```diff
@@ -28,7 +28,7 @@ fn from_rpc_block(block: &RPCBlock) -> HeaderParams {
     HeaderParams {
         beneficiary: Address::from_str(&block.miner).unwrap(),
         timestamp: U256::from_str(&block.timestamp).unwrap().as_u64(),
-        number: U256::from_str(&block.number).unwrap(),
+        number: U256::from_str(block.number.as_ref().unwrap()).unwrap(),
```

Comment:
`unwrap` without returning `Result` or any error-check will produce runtime fails. Bad practice.

Reply:
Been like that in original code, with an exception of the fact that fail would happen at `U256::from_str` or down the line because of null value in `block.number`.